doc/admin/deployment.md

```diff
@@ -810,7 +818,7 @@ man page for additional options.
 To start the DAOS server, run:
 
 orterun -np &lt;num\_servers&gt; --hostfile \${hostfile}
---enable-recovery --report-uri \${urifile} daos\_server
+--enable-recovery --report-uri \${urifile} daos\_server -i
```

Comment:
Should probably note what "-i" does above before asking folks to use it. This overrides the allow_insecure line in the config file, right?